diff --git a/releng/pacman.conf b/releng/pacman.conf
index e7e6497..5a30d87 100644
--- a/releng/pacman.conf
+++ b/releng/pacman.conf
@@ -30,7 +30,7 @@ Architecture = auto
 
 # Misc options
 #UseSyslog
-#Color
+Color
 #NoProgressBar
 # We cannot check disk space from within a chroot environment
 #CheckSpace
@@ -38,6 +38,7 @@ Architecture = auto
 ParallelDownloads = 5
 #DownloadUser = alpm
 #DisableSandbox
+ILoveCandy
 
 # By default, pacman accepts packages signed by keys that its local keyring
 # trusts (see pacman-key and its man page), as well as unsigned packages.
